source/tutorial/react-native.txt

```diff
@@ -287,18 +294,15 @@ directory:
    * - ``ItemSchema.tsx``
        
      - The ``Item`` class, including its object data model. We import this
-       class in ``RealmContext.ts`` to include it in the app's overall Realm
+       class in ``AppWrapper.tsx`` to include it in the app's overall Realm
        schema.
      
-   * - ``RealmContext.ts``
```

Comment:
I notice you're removing RealmContext from here - should `AppWrapper.tsx` have an entry in this table?

Reply:
Good catch. I added it, with some info.